### ISSUE TITLE
.. include:: <test.txt>    :literal: not being included

### ISSUE THREAD

**Issue by ashwinkumar12345** [sphinx-doc/sphinx#2161]
Show code example for onclick button function is not being included from the example folder using 
.. include:: <test.txt>    :literal: 

Any reasons for make html not including the text files in html?

**Comment by tk0miya** [comment]
Right, the source code which is indicated with "Show source" does not contains included text.
At this moment, it is only copied from original .rst file.

+1 for improvement.
